notebooks/3_KernelMethods.ipynb

```diff
@@ -133,7 +137,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "K_train_raw = kernel_func(X_train, X_train)"
+    "K_train_raw = kernel_func(X_train, X_train, gamma = 1.0)"
```

Comment:
Do you center these?

Reply:
It will be centered later.
![image](https://user-images.githubusercontent.com/56018055/99823874-32d58280-2b55-11eb-980e-6d4fd88b476a.png)